doc/languages-frameworks/beam.section.md

```diff
@@ -280,6 +280,39 @@ mkShell {
 }
 ```
 
+### Using an overlay
+
+If you need to use an overlay to change some attributes of a derivation, e.g. if you need a bugfix from a version that is not yet available in nixpkgs, you can override attributes such as `version` (and the corresponding `sha256`)…
+
+#### `nix/elixir-1.13.1.nix`
+
+```nix
+self: super:
+{
+  elixir_1_13 = super.elixir_1_13.override {
+    version = "1.13.1";
+    sha256 = "0z0b1w2vvw4vsnb99779c2jgn9bgslg7b1pmd9vlbv02nza9qj5p";
+  };
+}
+```
+
+… and then use this overlay in your development environment:
+
+#### `shell.nix`
+
+```nix
+let
+  elixir_1_13_1_overlay = import ./nix/elixir-1.13.1.nix;
```

Comment:
hey, this is amazing! Thanks a lot for the contribution!

I have just one minor nit. Would you mind combining the 2 code snippets together ? So as to just have a shell.nix.
I'm don't feel very strong about this, I just feel it would be more concise, and that separating doesn't add too much value.
If you feel very strongly about this, let me know, I'm happy to merge as is.

Reply:
No, that's a good suggestion. Will update the PR. 👍🏻